lib/ansible/modules/source_control/git.py

```diff
@@ -977,6 +999,17 @@ def main():
     track_submodules = module.params['track_submodules']
 
     result.update(before=None)
+
+    if archive_fmt not in ['zip', 'tar']:
+        archive_fmt = 'tar'
+
+    if archive:
+        cmd = "%s archive --format=%s --remote=%s --output=%s master" \
```

Comment:
Should this command not be using a variable for the branch, in the case that it is specified by the `version` argument in a playbook?